_includes/desktop-update.md

```diff
@@ -2,10 +2,8 @@
 
 Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can choose when to start the download and installation process.
 
-> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder by clicking **Snooze**.
+> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder by clicking **Snooze**. All users can now skip an update when a reminder appears by clicking the **Skip this update** option.
```

Comment:
Do we really need to emphasize that’s it’s now all users?
Can’t we just say something like
```suggestion
> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder by clicking **Snooze**. You can skip an update when a reminder appears by clicking the **Skip this update** option.
```
?

Reply:
Yes; was thinking that as well 👆

Reply:
Yea, I was't sure, but thought it highlighted the change for existing users. Perhaps this is unnecessary background info for new users?

Reply:
Updated